### PR TITLE
added true,pred,all normalisation options to confusion matrix plot

### PR DESCRIPTION
**Pull request by hugocool**
these changes allow the practitioner to select how to normalize the confusion matrix plot
Screenshot 2021-06-14 at 17 57 01

### CODE REVIEW

**Comment by oegedijk** [comment]
Thanks so much for the contribution! Looks good!

Although I would rename the normalization option "true" to "observed" to be consistent with the graph and other nomenclature in the dashboard...

File: explainerdashboard/explainer_plots.py [hunk 1]
```diff
@@ -820,7 +820,7 @@ def plotly_dependence_plot(X_col, shap_values, interact_col=None,
 
 def plotly_shap_violin_plot(X_col, shap_values, X_color_col=None, points=False, 
         interaction=False, units="", highlight_index=None, idxs=None, round=3,
-        cats_order=None, max_cat_colors=5):
+        cats_order=None, max_cat_colors=5):  # sourcery no-metrics
```
Comment (oegedijk) [inline explainerdashboard/explainer_plots.py:823]: Not sure what this comment is about?

**Comment by oegedijk** [comment]
Seems a few tests need to be updated as well to replace the` normalized` parameter to `plot_confusion_matrix()` to `percentage`...

File: explainerdashboard/explainer_plots.py (hunk 1 above)
Comment (hugocool) [reply explainerdashboard/explainer_plots.py:823]: thats an artifact from [sourcery](https://sourcery.ai/), my apologies.

**Comment by oegedijk** [comment]
will have a more detailed look next week once I'm back from holiday, and then do the release, but looks good!

**Comment by oegedijk** [comment]
Alright, merged it! Thanks for the contribution! Will release it shortly depending on some other stuff I'm working on (static html export)...

thanks again!

**Comment by hugocool** [comment]
Sounds good! 🙏🏻 
There are some more contributions I would like to make, some i presume are well within the scope of explainerdashboard (like adding the violin and compact_dot plot options to the detailed summary plot). Or exposing the div id’s of the dashboard component to incorporate intro.js (or musicfox’s implements called mdc.Hello). 
While others might be out of scope for this package (like embedding an API call for bentoML prediction  (instead using the host of the explainerdashboard for prediction).

Should I make separate pull requests/issues for in-scope issues and move discussion of potentially out-of-scope suggestions elsewhere? If yes, where would be preferred?

**Comment by oegedijk** [comment]
Sounds good! 

I think best thing to do is open a seperate issue and then a PR for each contribution. That way others can follow the discussion and may be able to pitch in. 

Including specific support for bentoML might be a bit too specific, but perhaps there could be a general API of which bentoML would be one usecase?

**Comment by oegedijk** [comment]
included in the latest release!

**Pull request merged**